src/sign/index.ts

```diff
@@ -1,10 +1,6 @@
 import { createHmac } from "crypto";
 
 export function sign(secret: string, payload: string | object): string {
-  if (!secret || !payload) {
-    throw new TypeError("secret & payload required");
-  }
-
```

Comment:
I think we should leave this in. Removing it is a breaking change, albeit a small one. Maybe add above

```ts
// @ts-ignore throw friendly error message when required options are missing
```

Reply:
Done.